app/src/main/kotlin/com/hippo/ehviewer/ui/screen/DownloadsScreen.kt

```diff
@@ -648,7 +648,9 @@ fun AnimatedVisibilityScope.DownloadsScreen(navigator: DestinationsNavigator) =
     ) {
         if (!selectMode) {
             onClick(Icons.Default.Shuffle) {
-                withUIContext { navToReader(list.random().galleryInfo) }
+                if (list.isNotEmpty()) {
```

Comment:
Maybe we just should not show that button if list is empty? now that we have efficient dsl.

Reply:
Well, other buttons (sort, filter) are effectively no-op as well when the list is empty.